en/config_fw/position_tuning_guide_fixedwing.md

```diff
@@ -1,13 +1,8 @@
-# Advanced Fixed-wing Position Tuning
+# Fixed-wing Position Tuning
```

Comment:
The summary calls this "Fixedwing Altitude/Position Controller Tuning Guide". Which is more accurate?

The more accurate version should be here. The more terse version should be in the sidebar if they can't be the same.

Reply:
Renamed both to "Fixed-wing Altitude/Position Controller Tuning"